uinit/main.go

```diff
@@ -5,42 +5,72 @@ import (
 	"log"
 	"os/exec"
 	"time"
-)
 
-// TODO allow to specify boot sequence
+	"github.com/insomniacslk/systemboot/pkg/booter"
+)
 
 var (
-	doQuiet  = flag.Bool("q", false, "Disable verbose output")
-	interval = flag.Int("I", 1, "Interval in seconds before looping to the next boot command")
+	doQuiet       = flag.Bool("q", false, "Disable verbose output")
+	interval      = flag.Int("I", 1, "Interval in seconds before looping to the next boot command")
+	noDefaultBoot = flag.Bool("nodefault", false, "Do not attempt default boot entries if regular ones fail")
 )
 
-var bootsequence = [][]string{
+var defaultBootsequence = [][]string{
 	[]string{"netboot", "-userclass", "linuxboot"},
 	[]string{"localboot"},
 }
 
+var supportedBooterParsers = []func([]byte) (booter.Booter, error){
```

Comment:
Used for what ?

Reply:
On mobile now - however it's an interface that all the New* functions that return a Booter have to implement

Reply:
checked on this again - it's removed in a local commit since it's duplicated from another file, it will go away. I was implementing tests which is why I didn't push that yet